icubam/__init__.py

```diff
@@ -0,0 +1,14 @@
+import pkg_resources
+from os import path
+from setuptools.config import read_configuration
+
+def _extract_version(package_name):
+    try:
+        return pkg_resources.get_distribution(package_name).version
+    except pkg_resources.DistributionNotFound:
+        _conf = read_configuration(path.join(
+            path.dirname(path.dirname(__file__)), 'setup.cfg')
+        )
+        return _conf['metadata']['version']
+
+__version__ = _extract_version('icubam')
```

Comment:
To move forward on this let's just put,
```suggestion
__version__ = '0.1.2'
```
(same as in resources/config.toml) for now and address this versioning issue in a separate PR closing https://github.com/icubam/icubam/issues/121